exchanges/binance/binance_test.go

```diff
@@ -1174,9 +1175,9 @@ func TestGetMostRecentTrades(t *testing.T) {
 func TestGetHistoricalTrades(t *testing.T) {
 	t.Parallel()
 
-	_, err := b.GetHistoricalTrades("BTCUSDT", 5, 0)
-	if !mockTests && err == nil {
-		t.Error("Binance GetHistoricalTrades() expecting error")
+	_, err := b.GetHistoricalTrades("BTCUSDT", 5, -1)
```

Comment:
This is returning a 404 if mock tests are enabled. Adding `b.HTTPRecording = true` and turning mock tests off will record the response. This is not stopping a merge though.

Reply:
@yangrq1018 could you please address this and then this looks good to be merged!

Reply:
I get it! The response is recorded and push to the json file. The mock and off-mock tests are passed. Will markt it resolved.